universities/university-of-zagreb.md

```diff
@@ -0,0 +1,35 @@
+# University of Zagreb
+
+- **Website:** [University of Zagreb](https://www.unizg.hr)
+- **Open Science Policy:** [Open Science at UNIZG](https://interoperable-europe.ec.europa.eu/collection/open-source-observatory-osor/news/open-source-service-open-science)
+- **Centre for Software Development:** [Software Development at UNIZG](https://www.foi.unizg.hr/en/about-us/departments/czrpp)
+
+## Overview  
+
+The University of Zagreb (UNIZG) is Croatia’s largest and oldest university, recognized for its research initiatives in technology, informatics, and interdisciplinary sciences. While UNIZG actively promotes open science and open source collaboration, it **does not currently have a formal Open Source Program Office (OSPO).** However, it has taken steps to integrate open source principles into its research and development ecosystem.  
+
+## Open Source Science Collaboration  
+
+UNIZG participates in and supports initiatives that promote open source and open science practices:  :  
+
+- **Open Science Policy:** In 2024, UNIZG adopted an Open Science Policy aimed at improving accessibility and transparency in research. The policy encourages researchers to store original program code in open repositories like GitHub to foster reproducibility.  
+  [Read about Open Science Policy](https://interoperable-europe.ec.europa.eu/collection/open-source-observatory-osor/news/open-source-service-open-science)
+
+- **Centre for Software Development:** Housed at the Faculty of Organization and Informatics, this center actively monitors and participates in open source software projects. It provides resources and support for students and researchers engaged in open development.  
+  [Explore Software Development at UNIZG](https://www.foi.unizg.hr/en/about-us/departments/czrpp)
+
+## Contact
+
+For inquiries related to open source activities at the University of Zagreb, please contact:
```

Comment:
I thjink just leaving the websites works. "You can find their details on these pages:"